bytewriter.go

```diff
@@ -0,0 +1,68 @@
+package vfsgen
+
+import (
+	"fmt"
+	"io"
+	"strings"
+)
+
+// byteWriter encodes the input as a []byte literal.
+type byteWriter struct {
+	w     io.Writer
+	ntabs int
+	err   error
+}
+
+func (r *byteWriter) Write(p []byte) (n int, err error) {
+	for len(p) > 0 {
+		s := 16
```

Comment:
I would attempt to make this function more comprehensible for an average reader (like myself 😄). I.e. what is "s"? Why is it equal 16? What is indent and why is it needed? And comments before each "if" branch would be helpful too.